Automattic/wp-calypso
# Migrate Whoops Image Uses to Webpack

#### Changes proposed in this Pull Request

This migrates the three image uses of the "whoops" illustration to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be three uses of this image: https://github.com/Automattic/wp-calypso/search?q=%22whoops.svg%22&unscoped_q=%22whoops.svg%22

One usage is visiting a Portfolio page on a site which you're not an administrator on.

<img width="1039" alt="Screenshot 2020-01-13 at 20 10 57" src="https://user-images.githubusercontent.com/43215253/72288411-e968fc80-3640-11ea-9832-fa4b1ea25aac.png">

I recognise this isn't ideal, but I was hoping for some help on how to see the other two usages? It shouldn't be necessary to see them though, checking the code _should_ be enough.
